eng/spotbugs-aggregate-report/pom.xml

```diff
@@ -27,11 +27,14 @@
     <azure-identity.version>1.0.0-preview.3</azure-identity.version>
     <azure-keyvault.version>4.0.0-preview.3</azure-keyvault.version>
     <azure-messaging-eventhubs.version>5.0.0-preview.3</azure-messaging-eventhubs.version>
-    <azure-messaging-eventhubs-checkpointstore-blob.version>1.0.0-preview.1</azure-messaging-eventhubs-checkpointstore-blob.version>
+    <azure-messaging-eventhubs-checkpointstore-blob.version>1.0.0-preview.1
+    </azure-messaging-eventhubs-checkpointstore-blob.version>
```

Comment:
Line wrap necessary?

Reply:
Could be wrapping due to a 120 character limit on the auto format functionality IntelliJ offers, will bump that up for XMLs